src/main/java/com/google/cloud/genomics/dataflow/utils/ShardedBAMWritingOptions.java

```diff
@@ -0,0 +1,18 @@
+package com.google.cloud.genomics.dataflow.utils;
```

Comment:
Good point.

So perhaps for the case where a specific transform needs an option, that option should be in an inner class for the transform?  For example:
* KeyReadsFn.Options contains --lociPerShard
* The pipeline has an inner class `Options extends KeyReadsFn.Options, GenomicsOptions {. . .}`